content/docs/concepts/csidriver/features/powermax.md

```diff
@@ -709,3 +709,92 @@ managementServers:
+
+This feature supports the use of a StorageClass that is not associated with any specific PowerMax array or storage resource pool (SRP). Each cluster node must be labelled with topology labels which match the labels in the secret.
+
+Requirements:
+
+- Only one PowerMax array per availability zone is supported.
+- Every PowerMax array must be labelled with one or more zone labels.
+- Every cluster worker node must be assigned to a zone.
+- The volumeBindingMode must be set to WaitForFirstConsumer.
+
+> **Note:** Cluster nodes that are not in a zone will not be able to access arrays that are zoned. Conversely, arrays that are unzoned will only be accessible from cluster nodes that are unzoned.
+
+With this support a single storage class can be used to provision volumes from a pool of PowerMax arrays based on topology information where each AZ has its own PowerMax array. The following represents an example of the secret showing two arrays in different availability zones:
+
+### Secret
+
+```yaml
+storageArrays:
+  - storageArrayId: "000000000001"
+    primaryEndpoint: https://primary-1.unisphe.re:8443
+    backupEndpoint: https://backup-1.unisphe.re:8443
+    labels:
+      topology.kubernetes.io/region: region1
+      topology.kubernetes.io/zone: zone1
+    parameters:
+      SRP: srp_1
+      ServiceLevel: Gold
+  - storageArrayId: "000000000002"
+    primaryEndpoint: https://primary-2.unisphe.re:8443
+    backupEndpoint: https://backup-2.unisphe.re:8443
+    labels:
+      topology.kubernetes.io/region: region1
+      topology.kubernetes.io/zone: zone2
+    parameters:
+      SRP: srp_2
+```
+
+### Labelling Worker Nodes
+
+The worker nodes should have corresponding labels to match an availability zone.
+
+```bash
+# Label each worker node in the cluster
+kubectl label nodes worker-1 topology.kubernetes.io/region=region1
+kubectl label nodes worker-2 topology.kubernetes.io/region=region1
+...
+kubectl label nodes worker-1 topology.kubernetes.io/zone=zone1
+kubectl label nodes worker-2 topology.kubernetes.io/zone=zone2
+```
+
+### Storage Class
+
+When using multiple availability zones the StorageClass does not require details about the PowerMax array. The following sample shows a minimal StorageClass definition:
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: powermax-multi-az
+parameters:
+  csi.storage.k8s.io/fstype: xfs
+provisioner: csi-powermax.dellemc.com
+volumeBindingMode: WaitForFirstConsumer
+reclaimPolicy: Delete
+```
+
+With the above configuration, nodes labelled with the matching region, _region1_ and zone _zone1_ will access volumes provisioned on PowerMax array 000000000001. Nodes labelled with region _region1_ and zone _zone2_ will access volumes provisioned on PowerMax array 000000000002. Different storage resource pools (SRP) will be used for each array based on the specifications in the secret.
```

Comment:
So just wanted to clarify: if a storageArray in the secret has multiple labels, all labels of the secret must also exist on the node in order for the node to be allowed to mount the volume during the create volume procedure.

Reply:
Correct. All or nothing.